Gruntfile.js

```diff
@@ -292,10 +292,9 @@ module.exports = function(grunt) {
     },
 
     shell: {
-      'npm-install': {
-        command: 'node scripts/npm/check-node-modules.js'
+      'install-node-dependencies': {
+        command: 'yarn'
```

Comment:
I wonder why it's not that fast on the second try, though. I'd expect it to behave identically in all situations where cache is populated and dependencies are correct so something looks wrong here.